packages/edit-site/src/hooks/commands/use-wp-admin-commands.js

```diff
@@ -44,7 +44,7 @@ const getWPAdminAddCommandLoader = ( postType ) =>
 		const commands = useMemo(
 			() => [
 				{
-					name: 'core/wp-admin/add-' + postType,
+					name: 'core/wp-admin/add-' + postType + '-' + search,
```

Comment:
The command menu package we use, rely on the "value" prop (which is the name of the command for us) to actually filter the results. This command was returned by this hook but was hidden by the search algorithm.